tests/integration_tests/modules/test_power_state_change.py

```diff
@@ -64,7 +64,7 @@ class TestPowerChange:
         [
             ("poweroff", "now", "10", "will execute: shutdown -P now msg"),
             ("reboot", "now", "0", "will execute: shutdown -r now msg"),
-            ("halt", "+1", "0", "will execute: shutdown -H +1 msg"),
+            ("halt", "+1", "0", re.escape("will execute: shutdown -H +1 msg")),
```

Comment:
Did this ever pass?

Perhaps something changed in the verify ordering utility.

Reply:
There's a good likelihood it never passed